doc/BACKUP.md

```diff
@@ -0,0 +1,413 @@
+PostgreSQL cluster to be used by `lightningd`, with automatic replication
+and turnover in case an entire node of the PostgreSQL cluster fails.
+
+Setting this up, however, is more involved.
+
+By default, `lightningd` compiles without PostgreSQL support.
+To enable it, you have to install a developer version of `libpq`.
+On most Debian-derived systems that would be `libpq-dev`.
+To verify you have it properly installed on your system, check if the
+following command gives you a path:
+
+    pg_config --includedir
+
+Versioning may also matter to you.
+For example, Debian Stable ("buster") as of late 2020 provides PostgreSQL 11.9
+for the `libpq-dev` package, but Ubuntu LTS ("focal") of 2020 provides
+PostgreSQL 12.5.
+Debian Testing ("bullseye") uses PostgreSQL 13.0 as of this writing.
+PostgreSQL 12 had a non-trivial change in the way the restore operation is
+done for replication.
+You should use the same PostgreSQL version of `libpq-dev` as what you run
+on your cluster, which probably means running the same distribution on
+your cluster.
+
+Once you have decided on a specific version you will use throughout, refer
+as well to the "synchronous replication" document of PostgreSQL for the
+**specific version** you are using:
+
+* [PostgreSQL 11](https://www.postgresql.org/docs/11/runtime-config-replication.html)
+* [PostgreSQL 12](https://www.postgresql.org/docs/12/runtime-config-replication.html)
+* [PostgreSQL 13](https://www.postgresql.org/docs/13/runtime-config-replication.html)
+
+You then have to compile `lightningd` with PostgreSQL support.
+
+* Clone or untar a new source tree for `lightning` and `cd` into it.
+  * You *could* just use `make clean` on an existing one, but for the
+    avoidance of doubt (and potential bugs in our `Makefile` cleanup rules),
+    just create a fresh source tree.
+* `./configure`
+  * Add any options to `configure` that you normally use as well.
+* Double-check the `config.vars` file contains `HAVE_POSTGRES=1`.
+  * `grep 'HAVE_POSTGRES' config.vars`
+* `make`
+* If you install `lightningd`, `sudo make install`.
+
+If you were not using PostgreSQL before but have compiled and used
+`lightningd` on your system, the resulting `lightningd` will still
+continue supporting and using your current SQLITE3 database;
+it just gains the option to use a PostgreSQL database as well.
+
+To set up a cluster for a brand new node, follow this (external)
+[guide by @gabridome][gabridomeguide].
+
+[gabridomeguide]: https://github.com/gabridome/docs/blob/master/c-lightning_with_postgresql_reliability.md
+
+The above guide assumes you are setting up a new node from scratch.
+It is also specific to PostgreSQL 12, and setting up for other versions
+**will** have differences; read the PostgreSQL manuals linked above.
+
+If you want to continue a node that started using an SQLITE3 database,
+use the [tool by @fiatjaf][fiatjafmcldsp] to convert your SQLITE3 database
+to a PostgreSQL database.
```

Comment:
Hmmm .. Are you confident with it ? Even @fiatjaf [does not seem to be](https://github.com/fiatjaf/mcldsp#beware-this-may-destroy-your-life-and-make-you-lose-all-your-money-use-at-your-own-risk).  If we are confident we are able to provide such a **crucial** tool it shoudl probably be part of the main repo...

Reply:
No, have never used it, will remove the link instead.